### ISSUE TITLE
error message is missleading

### ISSUE THREAD

**Issue by goern** [junaruga/rpm-py-installer#75]
if rpm-devel is missing, a misleading error message is issues:

```
    [ERROR] CMD: [/usr/bin/rpm --query rpm-devel --quiet], Return Code: [1] at [/tmp/pip-build-fnv3a91e/rpm-py-installer]
    Traceback (most recent call last):
      File "install.py", line 581, in <module>
        main()
      File "install.py", line 576, in main
        app.run()
      File "install.py", line 35, in run
        self._verify_system_status()
      File "install.py", line 165, in _verify_system_status
        raise InstallError(message)
    __main__.InstallError: Required RPM not installed: [rpm-devel].
    Install it by "dnf install rpm-devel".
```

**Comment by junaruga** [comment]
Which part of the message is misleading?
What message do you prefer for that?

**Comment by goern** [comment]
Well, the first line is referring to RPM and the last to DNF... ;)

**Comment by junaruga** [comment]
@goern Your opinion makes sense.
"the first line is referring to RPM and the last to DNF... ;)" should be fixed.

Yeah, I wanted to show users how to solve the error concretely in the error message.
Thought the solution might be `rpm -i pkg_name`, `yum install pkg_name` or `dnf install pkg_name`.

**Comment by junaruga** [comment]
@goern as I fixed this issue by PR #77, I would close this.
But feel free to reopen if there is something.
